### PR TITLE
Feature added - Preview Markdown

### PR DESCRIPTION
**Pull request by Publicker**
Because see (#77):

* Is cool see the markdown preview

* Show more clear what are you doing in markdown

This commit:

* Includes a new package -> ReactMarkdown

* Add a new switch setting, enable or able preview mode (also able from keyboard shortcut)

* Simple styles for the previewer

### Attachment
takenote-preview-markdown

### CODE REVIEW

File: src/styles/_previwer.scss [hunk 1]
```diff
@@ -0,0 +1,27 @@
+.previewer {
```
Comment (Sheing) [inline src/styles/_previwer.scss]: the file name should be `_previewer.scss`

File: src/styles/index.scss [hunk 2]
```diff
@@ -4,6 +4,7 @@
 @import 'app-sidebar';
 @import 'note-sidebar';
 @import 'editor';
+@import 'previwer';
```
Comment (Sheing) [inline src/styles/index.scss]: `previewer`

File: src/styles/_previwer.scss (hunk 1 above)
Comment (Publicker) [reply src/styles/_previwer.scss]: Ohh 😅

You are right!

**Comment by taniarascia** [comment]
Excellent, @Publicker! Thanks for getting this feature ready.

**Comment by Publicker** [comment]
My pleasure!

**Pull request merged**